Client/RecommendationClient.php

```diff
@@ -22,4 +22,13 @@ public function updateContent($contentId);
      * @return void
      */
     public function deleteContent($contentId);
+
+    /**
+     * @param int $userId YooChoose user ID
+     * @param int $scenarioId YooChoose scenario ID
```

Comment:
I kind of disagree with @sunpietro's comment here. The parameters are well named, and it is sufficient to understand what they're for. Saying that $userId is a user id is pointless. Same for "$locationId: Location ID".

Comments are only useful when they add something. Remember that they must be maintained :-)